.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 orbs:
   yarn: artsy/yarn@2.1.1
   hokusai: artsy/hokusai@0.7.2
-  node: artsy/node@0.1.0
+  node: artsy/node@dev:dc523d4428b2260798906013dc4d8221
```

Comment:
I think we can go ahead and deploy the node version update. The `yarn` orb also uses node though, so that's where we'll need the canary.